source/_integrations/lg_thinq.markdown

```diff
@@ -0,0 +1,136 @@
+---
+title: LG ThinQ
+description: Setup for LG ThinQ Integration.
+featured: true
+ha_category:
+  - Hub
+ha_release: "2024.10"
+ha_iot_class: Cloud Push
+ha_code_owners:
+  - '@LG-ThinQ-Integration'
+ha_config_flow: true
+ha_domain: lg_thinq
+ha_platforms:
+  - switch
+ha_dhcp: true
+ha_integration_type: integration
+---
+
+The LG ThinQ integration allows you to connect LG ThinQ devices to Home Assistant. The features of this integration include:
+
+- Control LG appliances as Home Assistant entities through the [LG ThinQ Connect API](https://thinq.developer.lge.com/ko/cloud/).
+- This feature works via LG ThinQ cloud; therefore, an internet connection is required.
+
+
+## Prerequisites
+
+- An internet connection and a [personal access token](https://connect-pat.lgthinq.com) are required to use the [LG ThinQ Connect API](https://thinq.developer.lge.com/ko/cloud/).
+
+
+### Personal Access Token (PAT)
+
+1. Access the **[personal access token](https://connect-pat.lgthinq.com)** page (requires an LG ThinQ account).
+2. Click the **ADD NEW TOKEN** button.
+3. Enter a new token name and select the following authorized scopes: 
+    - Permission to view all devices
+    - Permission to view all device statuses
+    - All device control rights
+    - All device event subscription rights
+    - All device push notification permissions
+4. Click the **CREATE TOKEN** button.
+5. Once all the steps are completed, you will see that a **PAT** has been generated.
+
+### MQTT
+
+The LG ThinQ integration connects to the LG ThinQ Cloud via MQTT and receives push status updates from the devices. This is a part of the LG ThinQ API implementation and is automatically connected through internal operations, so no additional settings are required.
+
+{% note %} MQTT connections are only supported in the Appliance feature.{% endnote %}
+
+
+## Setup
+
+To set up the integration, please follow these steps:
+
+{% include integrations/config_flow.md %}
+
+1. Enter the information to use LG ThinQ Connect API:
+   - The **Token Value** obtained through the PAT issuance process. (Required)
+   - The entry name.
+2. Choose the region (country).
+
+
+## Delete
+
+To remove the integration, please follow these steps:
+
+1. Go to the Home Assistant Integrations page.
+2. Select the LG ThinQ integration instance.
+3. Click on the options menu and choose Delete.
+
+{% note %} Deleting the integration will remove all entities associated with your LG ThinQ devices' features. {% endnote %}
```

Comment:
Not sure if this is LG specific

Reply:
Good point. we should add a general procedure that integration docs can point to, and then add the integration specific information only.

Reply:
Do you mean we don't need it here because it is not LG specific? 
Then let me remove it.

Reply:
Removed.